daemon/delete.go

```diff
@@ -157,11 +161,7 @@ func (daemon *Daemon) volumeRm(name string) error {
 	}
 
 	if err := daemon.volumes.Remove(v); err != nil {
-		if volumestore.IsInUse(err) {
-			err := fmt.Errorf("Unable to remove volume, volume still in use: %v", err)
-			return errors.NewRequestConflictError(err)
-		}
-		return fmt.Errorf("Error while removing volume %s: %v", name, err)
+		return errors.Wrap(err, "Unable to remove volume")
```

Comment:
LOL, I had it lowercase, but saw all other errors were with uppercase, so changed it; I'll update that